recipes/zarr/meta.yaml

```diff
@@ -0,0 +1,40 @@
+{% set version = "1.0.0" %}
+
+package:
+  name: zarr
+  version: {{ version }}
+
+source:
+  git_url: https://github.com/alimanfoo/zarr.git
+  git_tag: v{{ version }}
+
+build:
+  number: 0
```

Comment:
You can a line here with `script: python setup.py install --single-version-externally-managed --record record.txt` and remove both `bld.bat` and `build.sh` files.